test/runtests.jl

```diff
@@ -4,11 +4,12 @@ using Test
 const testdir = dirname(@__FILE__)
 
 tests = [
-	"AtomFeat/atomfeat",
-	"AtomGraph/atomgraph",
+	"AtomGraph/AtomGraph_tests",
```

Comment:
Do the names of the test-files have to be suffixed with `_tests`? Isn't it implicitly understood these are tests since they're in the `tests` directory?

Reply:
I personally like it that way because then you don't have files with the same name floating around, which can be confusing at a glance for me if e.g. I have both of them open in my IDE.